styles/layouts/_main-navigation.scss

```diff
@@ -44,15 +42,21 @@
 	}
 }
 
+.has-spaced-navbar-fixed-top {
```

Comment:
what does this fix?

Reply:
It fixes the incorrect default height of navbar